imap_processing/mag/l1a/mag_l1a.py

```diff
@@ -305,12 +318,40 @@ def generate_dataset(
         attrs=attribute_manager.get_variable_attributes("vector_attrs"),
     )
 
+    compression_flags = xr.DataArray(
+        single_file_l1a.compression_flags,
+        name="compression_flags",
+        dims=["epoch", "compression"],
+        attrs=attribute_manager.get_variable_attributes("compression_flags_attrs"),
+    )
+
+    direction_label = xr.DataArray(
+        direction.astype(str),
+        name="direction_label",
+        dims=["direction_label"],
+        attrs=attribute_manager.get_variable_attributes("direction_label"),
+    )
+
+    compression_label = xr.DataArray(
+        compression.astype(str),
+        name="compression_label",
+        dims=["compression_label"],
+        attrs=attribute_manager.get_variable_attributes("compression_label"),
+    )
+
     output = xr.Dataset(
-        coords={"epoch": epoch_time, "direction": direction},
+        coords={
+            "epoch": epoch_time,
+            "direction": direction,
+            "compression": compression,
+            "direction_label": direction_label,
```

Comment:
Could you elaborate on `data_vars`? I'm not sure what that means. I agree they don't belong in the coordinates, but that may be more of a philosophical objection than any real complaint

Reply:
This is something Tim and I was discussing. Yes, it's a philosophical question at the moment. I don't think CDF will care but there are two approaches that we are following and Tim pointed out that we may want to converge on one approach. We didn't have enough people to discuss that this month.

Reply:
Sounds good, in that case I will leave it as is and we can move it later if we decide to go with the data_vars approach. It would be nice to have a standard